src/components/StateMeta.js

```diff
@@ -59,6 +59,24 @@ function StateMeta({stateCode, data, timeseries}) {
   const growthRate =
     (Math.pow(lastConfirmed / prevWeekConfirmed, 1 / 7) - 1) * 100;
 
+  const percentageDescription = (percentageValue, description) => {
+    let confirmedCasesText 
+
+    if (percentageValue > 0 && percentageValue < 1)
+    {
+      confirmedCasesText = "For every 1000 confirmed cases";
+      percentageValue = percentageValue * 10
```

Comment:
For semantics sake, i wouldn't call it `percentageValue`  anymore, preferably return it here or use a more appropriate name